### PR TITLE
edit troubleshooting, add links

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

- renamed original troubleshooting.md to debugging-deletes.md
- moved content around in ^
- added parent troubleshooting.md to facilitate navigation from README
- moved link to doc site up in README

### CODE REVIEW

**Comment by Bradamant3** [comment]
n.b. related fixes to come in separate PRs

File: docs/debugging-deletes.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+Version 0.7.0 introduced the ability to delete backups. However, you may encounter an issue if you try to 
+delete the `heptio-ark` namespace. The namespace can get stuck in a terminating state, and you cannot delete your backups. 
+To fix:
+
+1. If you don't have it, [install `jq`][0].
+
+1. Run:
+    
+    ```bash
+    kubectl -n heptio-ark get backup -o json | jq -c -r $'.items[] | "kubectl -n heptio-ark patch backup/" + .metadata.name + " -p \'" + (({metadata: {finalizers: ( (.metadata.finalizers // []) - ["gc.ark.heptio.com"]), resourceVersion: .metadata.resourceVersion}}) | tostring) + "\' --type=merge"'
```
Comment (ncdc) [inline docs/debugging-deletes.md]: This lost the `bash<( .... )` that was wrapped around the entire long command

File: docs/debugging-deletes.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+for backups, schedules, restores, and the Ark config. We strongly recommend that you keep this configuration. 
+This approach can help prevent issues with deletes.
+
+## For the curious: why the error occurs
+
+The Ark team added the ability to delete backups by adding a **finalizer** to each
+backup. When you request the deletion of an object that has at least one finalizer, Kubernetes sets
+the object's deletion timestamp, which indicates that the object is marked for deletion. However, it does
+not immediately delete the object. Instead, the object is deleted only when it no longer has
+any finalizers. This means that something -- in this case, Ark, in this case -- must process the backup and then
```
Comment (ncdc) [inline docs/debugging-deletes.md]: in this case 2x

**Comment by skriss** [comment]
LGTM!

**Comment by ncdc** [comment]
LGTM

**Pull request merged**